website/docs/docs/collaborate/trust-signals.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Trust signal icons"
```

Comment:
@rpourzand confirming the name is 'trust signal icons' or 'trust signals'?

Reply:
Honestly, trust signal is more of a blanket statement for a number of things, including this.. I'd just call this - "Data Health Signals".